Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -683,21 +783,23 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
             }}
           />
         )}
-        {exportSkillModalVisible && (
+        {exportSkillModalInfo && (
           <ExportSkillModal
-            isOpen={exportSkillModalVisible}
-            projectId={projectId}
-            onDismiss={() => setExportSkillModalVisible(false)}
-            onSubmit={() => setExportSkillModalVisible(false)}
+            isOpen
+            projectId={exportSkillModalInfo.projectId}
+            onDismiss={() => setExportSkillModalInfo(undefined)}
```

Comment:
we dont need to explicity pass undefined when calling

Reply:
same reason to avoid type error